Rakefile

```diff
@@ -40,6 +40,26 @@ end
 
 desc "Use Racc & Ragel to regenerate parser.rb & lexer.rb from configuration files"
 task :build_parser do
+  def assert_dependency_version(dep_name, required_version, check_script)
+    version = `#{check_script}`
+    if !version.include?(required_version)
+      raise <<-ERR
+build_parser requires #{dep_name} version "#{required_version}", but found:
+
+    $ #{check_script}
+    > #{version}
+
+To fix this issue:
+
+- Update #{dep_name} to the required version
+- Update the assertion in `Rakefile` to match the current version
+ERR
+    end
+  end
+
+  assert_dependency_version("Ragel", "7.0.0.9", "ragel -v")
```

Comment:
@rmosolgo @gjtorikian Is there a reason to pin to this particular version of ragel?

I'm only able to find 6.9 in my [apt repos](https://packages.ubuntu.com/artful/ragel).

Reply:
It was found to be significantly faster back in https://github.com/rmosolgo/graphql-ruby/pull/737, but I didn't add the lock until here. Yeah, sorry, you have to get the "Development" build from http://www.colm.net/open-source/ragel/!